spec/Section 6 -- Execution.md

```diff
@@ -666,7 +672,9 @@ ResolveFieldValue(objectType, objectValue, fieldName, argumentValues):
 Note: It is common for {resolver} to be asynchronous due to relying on reading
 an underlying database or networked service to produce a value. This
 necessitates the rest of a GraphQL executor to handle an asynchronous execution
-flow.
+flow. In addition, for fields that have a return type that is a List type, each
+value in a collection of values returned by {resolver} may be retrieved
+asynchronously.
```

Comment:
Alternative wording suggestion:

```suggestion
flow. If the field is of a list type, each
value in the collection of values returned by {resolver} may itself be retrieved
asynchronously.
```

I don't like mixing a non-normative note with "may" (which is usually normative); but I think this is okay.

Reply:
Sounds good!